src/test/kotlin/graphics/scenery/tests/examples/compute/CustomVolumeManagerExample.kt

```diff
@@ -40,7 +41,7 @@ class CustomVolumeManagerExample : SceneryBase("CustomVolumeManagerExample") {
             ))
         volumeManager.customTextures.add("OutputRender")
 
-        val outputBuffer = MemoryUtil.memCalloc(1280*720*4)
+        val outputBuffer = ByteBuffer(1280*720*4)
```

Comment:
I love the conciseness of this. But it makes me nervous that the kool buffer types are named `ByteBuffer` and `FloatBuffer`, when `java.nio` also has classes with those names. Does the possibility of name clashes or reader confusion concern you? If so, you could rename the kool buffer types to something else like `KoolByteBuffer` and `KoolFloatBuffer`, or `Uint8Buffer` and `Float32Buffer`.

Reply:
It's coherent with the current way to allocate Arrays and Lists

`val ints = IntArray(2)`
`val ints = List(2) { i -> i + 1 }`

Reply:
Nice. As long as they are actually returning `ByteBuffer` and `FloatBuffer` objects, that's awesome.